geospatial/dted.ksy

```diff
@@ -0,0 +1,399 @@
+        size: 2
+        encoding: ASCII
+      - id: match_merge_version
+        type: str
+        size: 1
+        encoding: ASCII
+      - id: maintenance_date
+        type: date_yymm
+      - id: match_merge_date
+        type: date_yymm
+      - id: maintenance_description
+        type: str
+        size: 4
+        encoding: ASCII
+      - id: producer_code
+        type: str
+        size: 8
+        encoding: ASCII
+      - id: reserved_for_future_use3
+        type: str
+        size: 16
+        encoding: ASCII
+      - id: product_specification
+        type: str
+        size: 9
+        encoding: ASCII
+      - id:  product_specification_amendment_number
+        type: str
+        size: 1
+        encoding: ASCII
+      - id:  product_specification_change_number
+        type: str
+        size: 1
+        encoding: ASCII
+      - id:  product_specification_date
+        type: date_yymm
+      - id:  vertical_datum
+        type: str
+        size: 3
+        encoding: ASCII
+      - id:  horizontal_datum_code
+        type: str
+        size: 5
+        encoding: ASCII
+      - id:  digitizing_collection_system
+        type: str
+        size: 10
+        encoding: ASCII
+      - id:  compilation_date
+        type: date_yymm
+      - id:  reserved_for_future_use4
+        type: str
+        size: 22
+        encoding: ASCII
+      - id:  origin_of_data
+        type: coordinate_pair_ddmmss_sh
+      - id:  sw_corner_of_data
+        type: coordinate_pair_ddmmssh
+      - id:  nw_corner_of_data
+        type: coordinate_pair_ddmmssh
+      - id:  ne_corner_of_data
+        type: coordinate_pair_ddmmssh
+      - id:  se_corner_of_data
+        type: coordinate_pair_ddmmssh
+      - id:  clockwise_orientation_angle
+        type: angle_dddmmss_sh
+      - id:  latitude_interval
+        type: ascii_int_ssss
+      - id:  longitude_interval
+        type: ascii_int_ssss
+      - id:  number_of_latitude_lines
+        type: ascii_int_ssss
+      - id:  number_of_longitude_lines
+        type: ascii_int_ssss
+      - id:  partial_cell_indicator
+        type: str
+        size: 2
+        encoding: ASCII
+      - id:  reserved_for_nima_use_only
+        type: str
+        size: 101
+        encoding: ASCII
+      - id:  reserved_for_producing_nation_use_only
+        type: str
+        size: 100
+        encoding: ASCII
+      - id:  reserved_for_free_text_comments
+        type: str
+        size: 155
+        encoding: ASCII
+  acc_header:
+    seq:
+      - id: recognition_sentinel
+        contents: ACC
+      - id: absolute_horizontal_accuracy
+        type: ascii_int_ssss
+      - id: absolute_vertical_accuracy
+        type: ascii_int_ssss
+      - id: relative_horizontal_accuracy_of_product_in_meters
+        type: ascii_int_ssss
+      - id: relative_vertical_accuracy_or_product_in_meters
+        type: ascii_int_ssss
+      - id: reserved_for_future_use1
+        type: str
+        size: 4
+        encoding: ASCII
```

Comment:
Encoding can be moved into `meta`

Reply:
Cool! I didn't realize you could do that.